climada/hazard/centroids/centr.py

```diff
@@ -709,17 +696,15 @@ def from_csv(cls, file_path):
 
 
     @classmethod
-    def from_excel(cls, file_path, crs=DEF_CRS, var_names=None):
+    def from_excel(cls, file_path, sheet_name):
         """Generate a new centroids object from an excel file with column names in var_names.
 
         Parameters
         ----------
         file_name : str
             absolute or relative file path
-        crs : dict() or rasterio.crs.CRS, optional
-            CRS. Default: DEF_CRS
-        var_names : dict, default
-            name of the variables. Default: DEF_VAR_EXCEL
+        sheet_name : str
+            name of sheet in excel file containing centroid information
```

Comment:
which write_excel method exactly are you referring to? I left this optional because even some of the tests we have (e.g. for storm europe) don't stick to the default of having a "centroids" sheet, so I wasn't sure if there may be applications or users where another sheet name is used

Reply:
The important is that you define a clear default option. A user should not have to open the file in excel (or pandas) before being able to open it with CLIMADA.

ooo and I see there is no `write_excel` and `write_csv` methods now for centroids. I think these are then missing.. We should support read/write in general I would say.